docs/store.js

```diff
@@ -395,11 +395,13 @@ var LocalCalendarStorage = function(params) {
     
     self.endWork = function() {
         // Will decrement the counter and maybe fire an event.
-        
-        
+
         self.activeCalls -= 1;
+        
         if (self.activeCalls < 0) {
             console.error("No work has been started");
```

Comment:
This error message could be better.  e.g. "Number of active calls is negative." 

The entire point is that you don't really know if there's work happening or not because clearly the counting system is broken if you're getting negative numbers.